### PR TITLE
Update it.yaml

### PR DESCRIPTION
**Pull request by sfederici**
Complete Italian translation

### CODE REVIEW

File: languages/it.yaml [hunk 1]
```diff
@@ -1,152 +1,152 @@
 tw.gui.crashMessage.description:
   englishMessage: We are so sorry, but it looks like TurboWarp has crashed. Please refresh your page to try again.
   description: Message to inform the user that page has crashed.
-  message: null
+  message: Ci dispiace, sembra che TurboWarp sia andato in crash. Aggiorna la pagina per riprovare
```
Comment (GarboMuffin) [inline languages/it.yaml:4]: missing period?

**Review by GarboMuffin — CHANGES_REQUESTED** [review]
This seems pretty good to me, aside from one missing period

I would like to get some feedback from other Italian speakers first before merging this, however

File: languages/it.yaml (hunk 1 above)
Comment (sfederici) [reply languages/it.yaml:4]: Yes, the final period is missing.

**Comment by sfederici** [comment]
Excellent. When is the Italian translation expected to be available online?

**Comment by GarboMuffin** [comment]
Hopefully within a few days. There's some changes that need to be made in other places to allow translations to actually be loaded

**Comment by GarboMuffin** [comment]
Italian translation is live on staging (https://staging.turbowarp.org/), will be live on production probably tomorrow

**Comment by GarboMuffin** [comment]
Also some more messages were added that were previously hard-coded as English. Consider submitting a pull request for those if you have time

**Comment by sfederici** [comment]
> Italian translation is live on staging (https://staging.turbowarp.org/), will be live on production probably tomorrow

Excellent! Thanks!

**Pull request merged**